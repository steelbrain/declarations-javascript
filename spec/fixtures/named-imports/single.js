import { moduleName as anotherName } from 'file-moduleName'
anotherName()
