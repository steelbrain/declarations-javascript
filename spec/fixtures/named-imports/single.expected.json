[
  {
    "name": "anotherName",
    "position": {
      "start": {
        "line": 2,
        "column": 0
      },
      "end": {
        "line": 2,
        "column": 11
      }
    },
    "source": {
      "name": "moduleName",
      "filePath": "file-moduleName",
      "position": {
        "start": {
          "line": 1,
          "column": 23
        },
        "end": {
          "line": 1,
          "column": 34
        }
      }
    }
  }
]